src/Symfony/Component/Notifier/Bridge/SpotHit/README.md

```diff
@@ -6,12 +6,14 @@ Provides [Spot-Hit](https://www.spot-hit.fr/) integration for Symfony Notifier.
 #### DSN example
 
 ```
-SPOTHIT_DSN=spothit://TOKEN@default?from=FROM
+SPOTHIT_DSN=spothit://TOKEN@default?from=FROM&smslong=SMSLONG&smslongnbr=SMSLONGNBR
 ```
 
 where:
  - `TOKEN` is your Spot-Hit API key
  - `FROM` is the custom sender (3-11 letters, default is a 5 digits phone number)
+ - `SMSLONG` (optional) 0 or 1 : allows SMS messages longer than 160 characters
+ - `SMSLONGNBR` (optional) Allows to check the size of the long SMS sent. You must send the number of concatenated SMS as a value. If our counter indicates a different number, your message will be rejected.
```

Comment:
So this parameter defines if I allow to check? So this should be a bool, like the option before, right? And if I am right, how can I "send the number of concatenated SMS as a value"?

Reply:
No it is directly an integer. "3" would mean that the SMS cannot exceed 3 SMS, otherwise the API throws an error.

The text is translated directly from the official documentation. I just added the expected type for more clarity

Reply:
Ok, understand, my question is, if this should be configured at project level? I mean would you as a developer limit it to 3 messages all the time, or should this be set programmatically via the options for example on runtime?

SMSLONG is ok to define on project level IMHO, because you can allow or forbid this (regarding the costs e.g.)

Reply:
Ok that's a good point. Personally I use it at project level, as a guard. This option being very specific to this provider, I imagine that a need at runtime would not be done via this option but directly calculated in the code to be compatible whatever the provider

Reply:
Maybe we should rename the options then?

SMSLONG -> LONG
SMSLONGNBR -> MAX 

it is more speakable, WDYT?

Reply:
I agree, the name is not clear at all. I hesitated. I thought it was best to use the terms as requested and documented by SpotHit. Do you think it would be better to change this?

Reply:
@fabpot WDYT?

Reply:
Let's keep the name used by the provider.